WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift

```diff
@@ -0,0 +1,25 @@
+/// dependency container for the What's New / Feature Announcements scene
+extension WPTabBarController {
+
+    func makeWhatIsNewViewController() -> WhatIsNewViewController {
+        return WhatIsNewViewController(whatIsNewViewFactory: makeWhatIsNewView)
+    }
+
+    private func makeWhatIsNewView() -> WhatIsNewView {
+
+        var version = ""
+        if let versionNumber = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
+            version = NSLocalizedString("Version ", comment: "Description for the version label in the What's new page.") + versionNumber
```

Comment:
What do you think about making a `private enum` for all the `NSLocalizedString`s in this method? It'd make it more readable. For example, I almost missed the tacked on `+ versionNumber` here as it was kind of obfuscated by the string definition.

Reply:
Totally more readable! Thanks for pointing this out!